src/utils/SharedUtils.js

```diff
@@ -0,0 +1,163 @@
+export function serialize(obj, prefix) {
```

Comment:
Oh ok. So with that in mind, is me writing my own xmlhttprequest functions useful? Or should I just be using the jQuery ones?